### ISSUE TITLE
Level set expectations with VFS teams re: artifacts 

### ISSUE THREAD

**Issue by CherylEvans** [department-of-veterans-affairs/va.gov-team#50725]
## User Story
"As a Governance team member, I want to convey to our Collaboration Cycle Touchpoint review audience that our reviews are based on artifacts provided to us within our specific CC timeline guidance, and may not reflect the current state of the product, so that if VFS teams fix issues, or make changes to the product during the Collaboration Cycle process, prior to the touchpoint review, the review audience will understand that our review was based solely on the artifacts we were provided by the VFS teams prior to the meeting, and prior to any changes/fixes they may have made by the VFS team during the process."

Assignee:  @briandeconinck 
Peer Reviewer: @CherylEvans

## Description
In the Midpoint Review Mural workshop, we recognized that sometimes VFS teams don't share with us up to date artifacts, they sometimes don't share artifacts that would be valuable, and they sometimes share incomplete artifacts. In ticket #54090, we revised the list of artifacts we'd like to see at Midpoint Review, and in this ticket, we'll "level set" with VFS teams to inform them that our reviews are based solely on the artifacts we receive. 

The purpose of this ticket is to add verbiage to the ~~Design Intent, Midpoint Review, and Staging Review Moderator's Responsibilities scripts~~ Design Intent, Midpoint Review, and Staging Review code blocks to indicate that our Collaboration Cycle touchpoint reviews are based solely on artifacts provided to us within our specific CC timeline guidance, and may not reflect the current state of the product, as VFS teams continue to make changes/fixes during the process, up to and including the initial posting of artifacts that we have requested to complete our review.

_Note: The decision was made to change editing the script from editing the code blocks as we want to make sure VFS teams know this when they share the artifacts and not after the fact. If we still feel like it is important to add this verbiage to the script, please coordinate with the Assignee for ticket #54635 (and escalate via the #platform-governance-team channel) as these tickets will be worked on in parallel._

## Impacted Artifacts
~~[Design Intent Moderator's Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1748336706/Design+Intent+-+Moderator+Responsibilities)~~
~~[Midpoint Review Moderator's Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1821442052/Midpoint+Review+-+Moderator+Responsibilities)~~
~~[Staging Review Moderator's Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1819476255/Staging+Review+-+Moderator+Responsibilities)~~
Collaboration Cycle Kickoff - Moderator responsibilities - code blocks for DI, MPR and SR

## Tasks
- [x] Add verbiage to the Artifacts section of each code block for DI, MPR, and SR
   - The verbiage does not have to be the exact same for all 3 touchpoints. It may even make sense to change the verbiage for all 3 since there's different outcomes of each meeting (eg. feedback for DI and MPR vs. findings for SR)

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] VFS teams understands that Collab Cycle feedback is dependent on artifacts shared
- [x] VFS teams understand that findings documented are dependent on artifacts shared

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@CherylEvans please attempt to define the tasks for this ticket. Thank you!

**Comment by CherylEvans** [comment]
@shiragoodman How does this sound?

**Comment by shiragoodman** [comment]
@CherylEvans it's very thorough, complete and accurate, however I have concerns that it may be too long. We're already crunched for time at the DI, so my thoughts are that we'd just add 1 extra line stating, "our review is based off of artifacts shared within the Collaboration Cycle Request ticket and/or in today's Design Intent meeting."

We can add something similar for MPR and SR too. 

In addition, do you recall why you opened this ticket? Was there push back from a team or confusion? I see your POV for why we want to add this, but I also want to make sure we're not spending time on something that's not really necessary. Please let me know.

**Comment by CherylEvans** [comment]
@shiragoodman You have made great suggestions and I agree it needs to be shorter in length. I will change it to reflect your recommendations. :-) As for your second question, there have been quite a few instances where we have given fb based on artifacts we were provided based on our CC guidance timeframe, and unbeknownst to us, changes/updates were made which rendered our fb obsolete.  I think it is essential for the review audience to understand that our fb is based specifically on the artifacts we are provided during the touchpoint window that we've provided however, since we can't stop a team from making updates/changes up to the very day of the review, it may not be the most current fb.  Make sense?

**Comment by CherylEvans** [comment]
@shiragoodman How does this sound? "Our feedback is based solely on artifacts shared within the Collaboration Cycle Request ticket window timeframe. It may be less up-to-date depending on changes or updates to the product since then."  It is generic enough to be added to ALL 3 of the touchpoint scripts, but gets the message across clearly and succinctly.

**Comment by shiragoodman** [comment]
Note: this ticket and ticket #55517 should be worked on in parallel.

**Comment by shiragoodman** [comment]
Note: this ticket and ticket #55517 should be worked on in parallel.

**Comment by briandeconinck** [comment]
Draft level-setting statements added to:
- [DI code block](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2579005497/55517+-+Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities#Design-Intent-code-block)
- [MPR code block](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2579005497/55517+-+Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities#Midpoint-Review-code-block)
- [SR code block](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2579005497/55517+-+Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities#Staging-Review-code-block)

Currently, only the Staging Review code block includes the link out to the Platform website guidance on staging artifacts. I've added similar links to the DI and MPR sections for consistency, but they're probably less critical for those touchpoints since there's less information about those artifacts on Platform website. Happy to take those out if things are feeling too wordy -- just let me know.

**Comment by briandeconinck** [comment]
@CherylEvans This and #55517 are ready for review!

**Comment by CherylEvans** [comment]
@briandeconinck If I understand correctly, you linked the Artifacts guidance from 55517 to the Di, MPR, and SR code blocks for this ticket?

**Comment by briandeconinck** [comment]
@CherylEvans Yes, that's correct

**Comment by CherylEvans** [comment]
@briandeconinck Great!  Since that is the case, all code blocks contain the linked guidance.  :-)
